html/inc/user.inc

```diff
@@ -283,6 +283,10 @@ function show_user_info_private($user) {
             tra("Account keys"),
             "<a href=\"weak_auth.php\">".tra("View")."</a>"
         );
+        row2(
+            tra("Account Ownership"),
+            "<a href=\"account_ownership_form.php?$url_tokens\">Generate ownership proof</a>"
+        );
```

Comment:
I think that this should only be displayed if the keys exist for this feature.

Reply:
done 👍